README.md

```diff
@@ -94,14 +94,17 @@ This change configures your app to always load the most recent version of your a
 
 To let the CodePush runtime know which deployment it should query for updates against, perform the following steps:
 
-1. Open your app's `Info.plist` and add a new `CodePushDeploymentKey` entry, whose value is the key of the deployment you want to configure this app against (e.g. the Staging deployment for FooBar app). You can retreive this value by running `code-push deployment ls <appName>` in the CodePush CLI, and copying the value of the `Deployment Key` column which corresponds to the deployment you want to use. 
+1. Open your app's `Info.plist` file and add a new entry named `CodePushDeploymentKey`, whose value is the key of the deployment you want to configure this app against (e.g. the Staging deployment for FooBar app). You can retreive this value by running `code-push deployment ls <appName>` in the CodePush CLI, and copying the value of the `Deployment Key` column which corresponds to the deployment you want to use (see below). Note that using the deployment's name (e.g. Staging) will not work. That "friendly name" is intended only for authenticated management usage from the CLI, and not for public consumption within your app.
```

Comment:
I think the example is a little bit ambiguous as to whether it's describing a deployment or a key, so I'd rephrase it from:

> whose value is the key of the deployment you want to configure this app against (e.g. the Staging deployment for FooBar app)

to:

> whose value is the key of the deployment you want to configure this app against (e.g. the key for the Staging deployment of the FooBar app)